### DISCUSSION TITLE
Help for How to Setup Project

### DISCUSSION THREAD

**Discussion by LemonAndMint** [ppy/osu#20052] [Q&A] [answered]
I cloned the osu! for study the project. Some libraries, like osu.Framework didnt found by vscode. i tried to look at wiki for how to setup the project but didnt understand at all. Can someone help me? (Im sorry if i done something wrong.)

**Comment by frenzibyte** [comment, marked as answer]
[`osu.Framework`](https://github.com/ppy/osu-framework) is referenced via nupkg in `osu.Game.csproj`:
https://github.com/ppy/osu/blob/a4dc3fe41215a7a24a98280141177804e1d5718e/osu.Game/osu.Game.csproj#L39

**Reply by LemonAndMint** [reply]
Thank you!
